internal/integration/client_test.go

```diff
@@ -818,6 +819,87 @@ func TestClient_BulkWrite(t *testing.T) {
 			})
 		}
 	})
+	mt.RunOpts("bulk write with bypassEmptyTsReplacement", mtBulkWriteOpts, func(mt *mtest.T) {
+		mt.Parallel()
+
+		newOpts := func(option bson.D) *options.ClientBulkWriteOptionsBuilder {
+			opts := options.ClientBulkWrite()
+			err := xoptions.SetInternalClientBulkWriteOptions(opts, "addCommandFields", option)
+			require.NoError(mt, err, "unexpected error: %v", err)
+			return opts
+		}
+
+		marshalValue := func(val interface{}) bson.RawValue {
+			t.Helper()
+
+			valType, data, err := bson.MarshalValue(val)
+			require.Nil(t, err, "MarshalValue error: %v", err)
```

Comment:
Use `mt` and `require.NoError`.

```suggestion
			mt.Helper()

			valType, data, err := bson.MarshalValue(val)
			require.NoError(mt, err, "MarshalValue error")
```